### DISCUSSION TITLE
Restricting Schema Introspection only for authenticated clients

### DISCUSSION THREAD

**Discussion by PeerMohamedMydeenA** [ChilliCream/hotchocolate#6997] [Q&A] [answered]
Hi, 

New to Banana Cake Pop, Is there a way to restrict the browse schema to only authenticated clients ? 

Thanks !

**Comment by PascalSenn** [comment, marked as answer]
@PeerMohamedMydeenA  Checkout this video: https://www.youtube.com/watch?v=WRdnpO5Xe7c

**Reply by PeerMohamedMydeenA** [reply]
@PascalSenn - This helps 👍
